lib/fml_forms/form.rb

```diff
@@ -1,6 +1,6 @@
 module FML
   class Form
-    attr_reader :form, :title, :version, :fieldsets, :fields, :bodySystem, :dependent
+    attr_reader :form, :title, :id, :version, :fieldsets, :fields, :bodySystem, :dependent
```

Comment:
That is the goal.  No need to fix in this pull request, just something I
recalled as looking through this.

On Tue, May 19, 2015 at 11:16 AM, Bill Mill <notifications@github.com>
wrote:

> In lib/fml_forms/form.rb
> <https://github.com/adhocteam/fml/pull/6#discussion_r30610149>:
>
> > @@ -1,6 +1,6 @@
> >  module FML
> >    class Form
> > -    attr_reader :form, :title, :version, :fieldsets, :fields, :bodySystem, :dependent
> > +    attr_reader :form, :title, :id, :version, :fieldsets, :fields, :bodySystem, :dependent
>
> it hasn't been since pretty early on in its development, if that's a goal
> for the future we should move towards it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adhocteam/fml/pull/6/files#r30610149>.
>



-- 
Greg Gershman
Principal, Ad Hoc LLC
443-928-2961
http://adhocteam.us